### PR TITLE
[nan-439] fix model output

### PR DESCRIPTION
**Pull request by khaliqgant**
## Describe your changes
Display model inputs correctly

## Issue ticket number and link
NAN-439

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: packages/shared/lib/services/nango-config.service.ts [hunk 1]
```diff
@@ -101,7 +101,10 @@ function getFieldsForModel(modelName: string, config: NangoConfig): { name: stri
         return null;
     }
 
-    const modelData = config.models[modelName] || config.models[`${modelName.slice(0, -1)}`];
+    // if it is an array of models, we still need to be able to recognize it
+    const strippedModelName = modelName.replace(/[\W_]+/g, '');
```
Comment (bodinsamuel) [inline packages/shared/lib/services/nango-config.service.ts]: I don't get this regex what's the input like?

**Review by bodinsamuel — APPROVED** [review]
Looks okay, I'm missing a bit the context of the fix here

File: packages/shared/lib/services/nango-config.service.ts (hunk 1 above)
Comment (khaliqgant) [reply packages/shared/lib/services/nango-config.service.ts]: It should replace `Foo[]` to be `Foo` so the model can be found correctly. I updated the regex in either case so it is clearer and does just that.

Comment (bodinsamuel) [reply packages/shared/lib/services/nango-config.service.ts]: ah okay I forgot about this array syntax, makes sense 👌🏻

**Pull request merged**